test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -35,23 +39,23 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-attacher
-          image: gcr.io/gke-release/csi-attacher:v2.0.0-gke.0
+          image: gke.gcr.io/csi-attacher:v2.0.0-gke.0
```

Comment:
theres's a 2.1.1-gke.0 available now

Reply:
it's not yet used in the OSS driver. We can update it after it is bumped and soaks for external PD tests for a while

Reply:
It's already running in staging rc: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/overlays/prow-gke-release-staging-rc/kustomization.yaml#L14